src/styles/examples.scss

```diff
@@ -177,6 +182,10 @@ article {
     .rv-xy-plot__axis__tick__line {
       stroke: $rv-xy-plot-axis-font-color;
     }
+    -webkit-user-select: none;
```

Comment:
this feels a little bit weird to be placing on .chart? can you place in somethign more specific to your example?